astarte-device-sdk-derive/Cargo.toml

```diff
@@ -8,6 +8,13 @@
 name = "astarte-device-sdk-derive"
 version = "0.1.0"
 edition = "2021"
+license = "Apache-2.0"
+repository = "https://github.com/astarte-platform/astarte-device-sdk-rust"
+description ="Astarte Device SDK"
```

Comment:
Changes to `Macro implementation of #[derive(AstarteAggregate)]`

Reply:
Should we keep it more generic?
This crate could contain more derive macros in the future, not just `AstarteAggregate`.